pkgs/development/libraries/libbladeRF/default.nix

```diff
@@ -11,16 +11,24 @@ stdenv.mkDerivation rec {
     name = "libbladeRF_v${version}-checkout";
   };
 
-  buildInputs = [ pkgconfig libftdi libusb udev cmake git ];
+  buildInputs = [ pkgconfig cmake git libusb1 udev ];
 
-# XXX: documentation fails to build due to a "undeclared here" bug.
-#      requires pandoc in buildInputs also..
-# YYY: udev rule wont install to "/etc/udev/rules.d/88-nuand.rules"???
   configurePhase = ''
```

Comment:
You could replace this `configurePhase` with
```
cmakeFlags = [
  "-DCMAKE_BUILD_TYPE=Debug"
  "-DINSTALL_UDEV_RULES_OFF"
  "-DBUILD_BLADERF_CLI_DOCUMENTATION=OFF"
];
```
Then, you don't need to set a custom `configurePhase`; including `cmake` in the `buildInputs` will activate a `setupHook` that will take care of configure and build correctly.